pandas/tests/io/json/test_pandas.py

```diff
@@ -1671,3 +1671,15 @@ def test_to_s3(self, s3_resource):
         assert target_file in (
             obj.key for obj in s3_resource.Bucket("pandas-test").objects.all()
         )
+
+    @pytest.mark.parametrize(
```

Comment:
I think the comment around parametrization got lost in GH comments but in any case would rather parametrize on `nulls_fixture` + `pd.NA` if we don't think we can add `pd.NA` to the former for now

I don't think the current parametrization is super useful, so OK to just take the first case and put it in the body of the test

Reply:
>  parametrize on nulls_fixture + pd.NA

+1

Reply:
Couldn't find a way to add `pd.NA` to a pytest fixture without recreating the contents of the fixture entirely, so I've added them as separate tests.  Let me know if you were thinking of something else.

Note that `pd.NaT` is the only object that doesn't use its string representation when encoded from a label.